src/services/variables.ts

```diff
@@ -1,11 +1,19 @@
 import { fetchRequest } from './fetchRequest';
 import { sortBy } from 'lodash';
+import {VariableDataType} from "./types";
 
+// type VariableDataType = {
+//   // DataType: string,
+//   Description: string,
+//   ID: number,
+//   // GroupName: string,
+//   Name: string,
+// }
 
-export const searchVariables = async () => {
+export const searchVariables = async (): Promise<VariableDataType[]> => {
   const res = await fetchRequest(`/vehicles/GetVehicleVariableList?format=json`);
 
-  const variablesColl = await res['Results'].map((variableData) => {
+  const variablesColl = await res['Results'].map((variableData: {Description: string, Name: string, ID: number} ) => {
```

Comment:
to prevent using CamelCase in your project you can wrap API result this library https://www.npmjs.com/package/camelcase-keys-deep and using caseCase in the project